### PR TITLE
Add todo info for function check_dmesg_output

### PR DESCRIPTION
**Pull request by chloerh**
This function could be replaced with another function of master
branch of avocado. Add a todo here as a reminder.

Signed-off-by: Haijiao Zhao <haizhao@redhat.com>

### CODE REVIEW

**Comment by chloerh** [comment]
@pevogam Would you like to take a look?

**Review by pevogam — APPROVED** [review]
LGTM, if you want to keep it easier in the future registering an issue at github is another good way to do the same.

File: virttest/utils_sys.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 from avocado.utils import process
 
 
+# todo: check function in avocado.utils after the next LTS
```
Comment (Yingshun) [inline virttest/utils_sys.py:13]: I'm not quite sure but I think it should be in upper case. TODO Tree(extension of vscode) cannot search for comment tag todo by default.

Comment (pevogam) [reply virttest/utils_sys.py:13]: Alternatively we could also go for sphinx compatible `..todo:: ` but I think our approach shouldn't depend on the choice of IDE-s.

Comment (chloerh) [reply virttest/utils_sys.py]: @Yingshun @pevogam Thanks for the suggestions. I'm going to follow the tradition of 'todo's of avocado-vt which is to replace it with 'TODO'.

**Review by Yingshun — APPROVED** [review]

**Pull request merged**